src/NuGet.Clients/VisualStudioAPI/IPackageManagerProvider.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.VisualStudio
```

Comment:
Unrelated, but please change the project name and folder from VisualStudioAPI to `NuGet.VisualStudio` the project name should match the generated assembly